src/calculateRentalCost.js

```diff
@@ -28,7 +28,27 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let rentalCost = 0;
+
+  const rentalCostOneDay = 40;
+  const minDiscount = 20;
+  const maxDiscount = 50;
+  const minDaysForDiscount = 3;
+  const maxDaysForMaxDiscount = 7;
+
+  if (days < minDaysForDiscount) {
+    rentalCost = rentalCostOneDay * days;
+  }
+
+  if (days >= minDaysForDiscount && days < maxDaysForMaxDiscount) {
+    rentalCost = rentalCostOneDay * days - minDiscount;
+  }
+
+  if (days >= maxDaysForMaxDiscount) {
+    rentalCost = rentalCostOneDay * days - maxDiscount;
+  }
```

Comment:
Also, you calculate `rentalCostOneDay * days` three times. Use varaible for it.

```suggestion
  if (days < minDaysForDiscount) {
    return rentalCostOneDay * days;
  }

  if (days >= minDaysForDiscount && days < maxDaysForMaxDiscount) {
    return rentalCostOneDay * days - minDiscount;
  }

  return rentalCost = rentalCostOneDay * days - maxDiscount;
```